loom/graphics/gfxVectorRenderer.cpp

```diff
@@ -353,10 +353,18 @@ void VectorRenderer::destroyGraphicsResources()
 
 void VectorRenderer::initializeGraphicsResources()
 {
+    int flags = 0;
+
+#if GFX_OPENGL_CHECK
+    flags |= NVG_DEBUG;
+#endif
+
+    flags |= NVG_ANTIALIAS | NVG_STENCIL_STROKES;

```

Comment:
That was my idea as well, any API suggestions? I was thinking the ANTIALIAS one should be some nanovg/vector specific switch as we may want to support actual™ OpenGL multisample antialiasing later on.